samples/bluetooth/central_bas/hids_keyboard.ncs

```diff
@@ -0,0 +1 @@
n","value":[0,0],"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":2,"errorMessage":null},"local.server.11.13.15":{"instanceId":"local.server.11.13.15","uuid":"2908","name":"Report Reference","value":[1,1],"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":2,"errorMessage":null}},"errorMessage":null},"local.server.11.16":{"instanceId":"local.server.11.16","uuid":"2A4D","name":"Report","value":[0],"properties":{"broadcast":false,"read":true,"writeWoResp":true,"write":true,"notify":false,"indicate":false},"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":1,"expanded":true,"discoveringChildren":false,"children":{"local.server.11.16.17":{"instanceId":"local.server.11.16.17","uuid":"2908","name":"Report Reference","value":[1,2],"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":2,"errorMessage":null}},"errorMessage":null},"local.server.11.18":{"instanceId":"local.server.11.18","uuid":"2A4B","name":"Report Map","value":[5,1,9,6,161,1,133,1,5,7,25,224,41,231,21,0,37,1,117,1,149,8],"properties":{"broadcast":false,"read":true,"write":false,"notify":false,"indicate":false},"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":22,"expanded":true,"discoveringChildren":false,"children":{},"errorMessage":null},"local.server.11.19":{"instanceId":"local.server.11.19","uuid":"2A22","name":"Boot Keyboard Input Report","value":[0,0,0,0,0,0,0,0],"properties":{"broadcast":false,"read":true,"write":false,"notify":true,"indicate":false},"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":8,"expanded":false,"discoveringChildren":false,"children":{"local.server.11.19.20":{"instanceId":"local.server.11.19.20","uuid":"2902","name":"Client Characteristic Configuration","value":[0,0],"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":2,"errorMessage":null}},"errorMessage":null},"local.server.11.21":{"instanceId":"local.server.11.21","uuid":"2A32","name":"Boot Keyboard Output Report","value":[0],"properties":{"broadcast":false,"read":true,"writeWoResp":true,"write":true,"notify":false,"indicate":false},"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":1,"expanded":false,"discoveringChildren":false,"children":{},"errorMessage":null},"local.server.11.22":{"instanceId":"local.server.11.22","uuid":"2A4A","name":"HID Information","value":[1,1,0,3],"properties":{"broadcast":false,"read":true,"write":false,"notify":false,"indicate":false},"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":4,"expanded":true,"discoveringChildren":false,"children":{},"errorMessage":null},"local.server.11.23":{"instanceId":"local.server.11.23","uuid":"2A4C","name":"HID Control Point","value":[0],"properties":{"broadcast":false,"read":false,"writeWoResp":true,"write":false,"notify":false,"indicate":false},"readPerm":"open","writePerm":"open","fixedLength":true,"maxLength":1,"expanded":true,"discoveringChildren":false,"children":{},"errorMessage":null}}}}}
```

Comment:
Does it make sense to have this file here? For the testing section in the docs, we could also just point to the file in central_hids. But it might be needed for other purposes of course. :)

Reply:
At the first run, I had an idea to provide a "clear BAS" server here. But then I realized that it would look better to provide a real application. The only thing I would worry about if this file would be in HIDS example only is that we have to update the link if the example localisations change.